app/auth/profile/settings.tsx

```diff
@@ -58,6 +58,13 @@ const Settings = () => {
         },
         label: "Rediger profil",
       },
+      {
+        icon: "trash-outline",
```

Comment:
Can you change this icon to person-remove-outline instead?

Reply:
Jeg synes selv at trash-outline giver mere mening og giver en bedre visualisering af hvad funktionen gør, men hvis det er det du ønsker så ændre jeg det

Reply:
Havde en snak internt i min gruppe og vi synes at trash-outline giver mere mening